### DISCUSSION TITLE
Connect to Code Server's server from VS Code for Desktop

### DISCUSSION THREAD

**Discussion by SergioLuis** [coder/code-server#2402] [Q&A] [answered]
Hello!

Is it possible to connect to the Code Server's server (the backend it is running) from Visual Studio Code Desktop? My use case would be to just use my Desktop application when possible, then switch to the browser-based editor when on the go.

My Code Server setup is running on a Docker container if it matters.

**Comment by code-asher** [comment, marked as answer]
It's not possible unfortunately. The remote extensions download VS
Code's server backend and execute then connect to that.

So the only way to make this work would be to modify the remote
extensions to work with code-server but since they're closed source we
can't and we'd need to re-implement them entirely.

There's no timeline for it at the moment but tracking for
re-implementing open source versions of those extensions can be found
here: https://github.com/cdr/code-server/issues/1315

For now I think the only workaround is to run both separately. You could
set code-server's user data directory to the one VS Code remote uses in
order to share plugins and settings.

**Reply by SergioLuis** [reply]
Thank you so much 👍

**Comment by slabach** [comment]
hey, I know I'm kind of a random here. But I just figured this out this morning....

you can [install PWA's](https://i.imgur.com/PXHGkxj.png) like code-server as app's using chrome. Linked an image, of the button in the url bar in chrome to 'install' that page. That gives you basically a native-like VS Code experience, entirely separate from your browser. Then on the go, you just go to your site like normal.

**Reply by neutronstriker** [reply]
Thanks, this helps!

**Comment by thienpow** [comment]
try something like this if you are on macOS
/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --new-window --app=http://192.168.100.8:8078/\?folder\=/home/coder/projects/manger
